OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-09-01/document.csv

```diff
@@ -2,14 +2,14 @@
 "Ta Mnr. D. F. Corlett[3] jeh Balleylaagh ceau tammylt beg er yn ellan ec y traa t’ayn. She yn Reiltagh eshyn jeh’n Cheshaght Unnanit Troggalys, feu Pt. 100,000,000, t’ayns Yn Afrika Jiass. Ta’n ven echey marishyn. Ta Mnr. Corlett smooinaghtyn dy vel Mannin foast jeeaghyn chammah as v’ee rieau.","     Mr D. F. Corlett of Ballaugh is spending a little while on the island at the moment. He is the President of the United Building Association, worth £ 100,000,000, in South Africa. His wife is accomanying him. Mr. Corlett thinks that the Isle of Man is still looking as good as it ever was.","[3] D. F. Corlett] Dan Corlett, born in Ballaugh, emigrated to South Africa in 1897. Was elected President of the Johannesburg  Master Builders Association in 1916, Served as mayor of Johannesburg 1931-1932. Served as President of the Transvaal Manx Association. Served as a member of the Board of the United Building Society (merged into the ABSA Group in 1986) from 1924, and as Chairman from 1938. Died in 1959."
-"Ta Manninagh elley er jeet thie nish myrgeddyn, ta shen dy ghra, Mnr. B. A. Corkill voish ny meainyn Rio Tinto ayns Yn Spainey, raad t’eshyn yn ard-injaneyr. ","     Another Manxman has come home now also, which is to say, Mr. B. A. Corkill from the Rio Tinto mines in Spain, where he is the shief-engineer.",
-"Daag Mnr. Corkill yn ellan e ghooie daa vlein as daeed er dy henney, as neayr’s shen ta’n obbyr echey er chur-lesh eh hug feed cheer.","Mr. Corkill left his native island forty-two years ago, and since them his work has brought him to twenty countries.",
+"Ta Manninagh elley er jeet thie nish myrgeddyn, ta shen dy ghra, Mnr. B. A. Corkill voish ny meainyn Rio Tinto ayns Yn Spainey, raad t’eshyn yn ard-injaneyr. ","     Another Manxman has come home now also, which is to say, Mr. B. A. Corkill from the Rio Tinto mines in Spain, where he is the chief-engineer.",
+"Daag Mnr. Corkill yn ellan e ghooie daa vlein as daeed er dy henney, as neayr’s shen ta’n obbyr echey er chur-lesh eh hug feed cheer.","Mr. Corkill left his native island forty-two years ago, and since then his work has brought him to twenty countries.",
 "Nee yn Cheshaght Kiaullee Streeleyryn dy gheiney, leeideilit liorish Vincent Shimmin, as yn Cheshaght Kiaullee Laureston, dy mreih, leeideilit liorish Lilian Pickard, ny neesht cohirrey ’sy Faailley Kiaullee ec Balley Dubbey-dhoo.","     The Wanderers Male Voice Choir, led by Vincent Shimmin, and the Laureston Ladies’ Choir, led by Lilian Pickard, will both compete in the Music Festival at Blackpool.",
-"Er y chiaghtyn shoh chaie, haink shin nyn guaiyl rish fer yindyssagh voish Baile Atha Claith.","     In the last week, we met a wonferfull fellow from Dublin. Thomas Ó Cleirigh was his name.",
+"Er y chiaghtyn shoh chaie, haink shin nyn guaiyl rish fer yindyssagh voish Baile Atha Claith.","     In the last week, we met a wonderful fellow from Dublin. Thomas Ó Cleirigh was his name.",
```

Comment:
Confirmed that this was not transcribed from the original document (original is Manx-only). Good to go!